### PR TITLE
improve demo scripts

### PR DESCRIPTION
**Pull request by zhreshold**
Use cpu by default

### CODE REVIEW

File: scripts/detection/faster_rcnn/demo_faster_rcnn.py [hunk 1]
```diff
@@ -8,14 +8,16 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Test with Faster RCNN networks.')
-    parser.add_argument('--network', type=str, default='faster_rcnn_resnet50_v1b_voc',
+    parser.add_argument('--network', type=str, default='faster_rcnn_resnet50_v1b_coco',
                         help="Faster RCNN full network name")
     parser.add_argument('--images', type=str, default='',
                         help='Test images, use comma to split multiple.')
-    parser.add_argument('--gpus', type=str, default='0',
+    parser.add_argument('--gpus', type=str, default='',
```
Comment (42B) [inline scripts/detection/faster_rcnn/demo_faster_rcnn.py]: Are `type=str` and `default=''` even necessary? Aren't those the defaults anyways?

File: scripts/detection/yolo/demo_yolo.py [hunk 2]
```diff
@@ -8,11 +8,11 @@
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Test with YOLO networks.')
-    parser.add_argument('--network', type=str, default='yolo3_darknet53_voc',
+    parser.add_argument('--network', type=str, default='yolo3_darknet53_coco',
                         help="Base network name")
     parser.add_argument('--images', type=str, default='',
                         help='Test images, use comma to split multiple.')
-    parser.add_argument('--gpus', type=str, default='0',
+    parser.add_argument('--gpus', type=str, default='',
```
Comment (42B) [inline scripts/detection/yolo/demo_yolo.py]: Same here?

**Comment by mli** [comment]
Job PR-449-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-449/1/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/detection/faster_rcnn/demo_faster_rcnn.py (hunk 1 above)
Comment (zhreshold) [reply scripts/detection/faster_rcnn/demo_faster_rcnn.py]: serves as extra guidance to user, doesn't hurt anyways.

**Pull request merged**